nucleus/__init__.py

```diff
@@ -48,6 +48,7 @@
 import tqdm
 import tqdm.notebook as tqdm_notebook
 
+from launch import LaunchClient
```

Comment:
would it make sense to have all the launch integration stuff inside some `nucleus/launch` subdirectory, s.t. nothing in `nucleus` imports from `nucleus/launch`, but `nucleus/launch` stuff imports from nucleus?

and then separately, I know you can theoretically do some `pip install scale-nucleus[launch-integration]`, although we still need to keep packages in sync between the two libraries

Reply:
if this is to prevent the requirements issues, i think i've gotten it to a relatively resolved state? with all the poetry wrangling lol